document-generator-accounts/src/main/java/uk/gov/companieshouse/document/generator/accounts/mapper/smallfull/model/ixbrl/balancesheet/currentassets/CurrentAssets.java

```diff
@@ -0,0 +1,91 @@
+package uk.gov.companieshouse.document.generator.accounts.mapper.smallfull.model.ixbrl.balancesheet.currentassets;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.gson.Gson;
+import uk.gov.companieshouse.document.generator.accounts.mapper.smallfull.model.ixbrl.balancesheet.currentassets.items.CashAtBankInHand;
+import uk.gov.companieshouse.document.generator.accounts.mapper.smallfull.model.ixbrl.balancesheet.currentassets.items.Debtors;
+import uk.gov.companieshouse.document.generator.accounts.mapper.smallfull.model.ixbrl.balancesheet.currentassets.items.Stocks;
+
+import java.util.Objects;
+
+public class CurrentAssets {
+
+    @JsonProperty("stocks")
+    private Stocks stocks;
+
+    @JsonProperty("debtors")
+    private Debtors debtors;
+
+    @JsonProperty("cash_at_bank_in_hand")
+    private CashAtBankInHand cashAtBankInHand;
+
+    @JsonProperty("current_total_current_assets")
```

Comment:
This field does not match small-full-accounts template. This should be `current_total`

Reply:
corrected these fields